lib/collection/src/lookup/types.rs

```diff
@@ -1,9 +1,33 @@
 use std::fmt::Display;
 
+use schemars::JsonSchema;
 use segment::data_types::groups::GroupId;
 use segment::types::PointIdType;
+use serde::{Deserialize, Serialize};
 use uuid::Uuid;
 
+use super::WithLookup;
+
+#[derive(Serialize, Deserialize, JsonSchema, Debug, Clone)]
+#[serde(untagged)]
+pub enum WithLookupInterface {
+    Collection(String),
+    WithLookup(WithLookup),
+}
+
+impl From<WithLookupInterface> for WithLookup {
+    fn from(with_lookup: WithLookupInterface) -> Self {
+        match with_lookup {
+            WithLookupInterface::Collection(collection_name) => Self {
+                collection_name,
+                with_payload: Some(true.into()),
+                with_vectors: Some(true.into()),
```

Comment:
usually, we don't return vectors by default.

Reply:
Defaulted to false now